### PR TITLE
[Feature:TAGrading] Fixed Columns for easy grading

### PR DESCRIPTION
**Pull request by immortalcodes**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
Adds the feature as requested in #7437 
Student column moves with the grid during grading,which casuse difficulties in grading as TA/Instructor have to go back and forth between the student column and the column  to mark and  make sure that they are marking the same row.

### What is the new behavior?
Now the student and other info columns remain fixed in the place which makes the grading easier.
image

### CODE REVIEW

File: site/public/css/simple-grading.css [hunk 1]
```diff
@@ -29,6 +29,49 @@ input[type="number"]::-webkit-outer-spin-button {
...
+  }
+tr>th:nth-child(5),tr>td:nth-child(5)
+ {
+    position: sticky;
+    left: 16.75rem;
+    border-right: 1px solid var(--standard-medium-gray);
+  }
+  
+  tr:nth-child(odd) td {
+    background: white;
```
Comment (williamjallen) [inline site/public/css/simple-grading.css:68]: Should this be fixed to white, or should it be changed to a variable listed in `colors.css` like the ones used elsewhere in these changes?

**Review by williamjallen — DISMISSED** [review]

**Review by shailpatels — CHANGES_REQUESTED** [review]
I tested this on windows firefox and chrome and got a strange issue where the animation and color of selecting a checkbox doesn't show anymore.

On main:
before

After installing the branc
after
h

In both browsers I can connect to the websockets server and I do not see any console errors. Are you able to check off students? I tested with a new gradeable created under the sample course

**Review by Eceptonsu — APPROVED** [review]
Works for me.

**Comment by tkoz0** [comment]
I'm seeing unexpected behavior with several checkpoints. In this image, I have selected checkpoint 1 with the arrow keys by moving left, but it is hidden behind the student information.
image
I also want to ask what the keyboard controls are supposed to be because I don't think it's clear or stated anywhere.

**Comment by Eceptonsu** [comment]
> I'm seeing unexpected behavior with several checkpoints. In this image, I have selected checkpoint 1 with the arrow keys by moving left, but it is hidden behind the student information. image I also want to ask what the keyboard controls are supposed to be because I don't think it's clear or stated anywhere.

The keyboard works for me. But one thing I noticed is that the title of the table is not at the center with respect to the window, it is located at the center of the entire table.
image
image

**Comment by immortalcodes** [comment]
Actually, the thing is that the fixed columns are overlapping the scrollable columns, this might not be entirely a problem since scrolling through the touchpad works well, there are possible solutions through javascript but looking for something simpler.

**Comment by immortalcodes** [comment]
image
I tried a lot of things but I guess the fix would look  something  like this.
In this, the upper title gets fixed on the left side of the table. 
The problem is with the column span which makes the title scrollable which otherwise would remain fixed.

**Review by shailpatels — APPROVED** [review]
Looks good, latest changes move the registration section name to the left side and not centered which I don't think is important since it remains visible the entire time

**Pull request merged**